source-facebook-marketing/source_facebook_marketing/schemas/videos.json

```diff
@@ -1,6 +1,9 @@
 {
   "type": "object",
   "properties": {
+    "account_id": {
+        "type": ["null", "string"]
```

Comment:
For these schemas, can the `account_id` ever really be an explicit `null`? I'd think it would either be present or absent, but always a string if it is present.

Reply:
You're correct. From what I've seen, `account_id` is either present as a string or absent, never an explicit `null`. I've been using `"type": ["null", "someOtherType"]` for fields that can be absent, regardless of whether they can be explicit `null`s since that's consistent with the other Airbyte imports. 

Would it be better to use `anyOf` like below instead? I _think_ this means the field is either present as a string or absent.
```python
"properties": {
    "account_id": {
        "anyOf": [
            { "type": "string"},
            { "type": "null"},
        ]
    }
}
```

Reply:
Present or not is controlled by the field being in the list of required fields. So a field that may either be present as a string or completely absent is `type: string` but is not in the list of required fields for the object.

Reply:
Got it, I see now. So `account_id` in these schemas can just be `"type": "string"`. I'll update those.